### PR TITLE
Add runbooks for KubeVirtVMStuckIn${STATE}State alerts

### PR DESCRIPTION
**Pull request by machadovilaca**
Create runbooks for alerts added in kubevirt/kubevirt/pull/7918

Signed-off-by: João Vilaça <jvilaca@redhat.com>

### CODE REVIEW

File: docs/runbooks/KubeVirtVMStuckInStartingState.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    Phase Transition Timestamp:   2022-10-03T11:08:30Z
+  Runtime User:                   0
+  Virtual Machine Revision Name:  revision-start-vm-6f01a94b-3260-4c5a-bbe5-dc98d13e6bea-1
+Events:
+  Type     Reason        Age                From                       Message
+  ----     ------        ----               ----                       -------
+  Warning  FailedCreate  8s (x13 over 28s)  virtualmachine-controller  Error creating pod: pods "virt-launcher-testvmi-ldgrw-" is forbidden: no PriorityClass with name non-preemtible was found
+```
+
+## Mitigation
```
[the remark this answers is not in the document]
Comment (iholder-redhat) [reply docs/runbooks/KubeVirtVMStuckInStartingState.md:68]: And to be more precise, as I see from [the code](https://github.com/kubevirt/kubevirt/blob/v0.58.0-rc.0/pkg/virt-controller/watch/vmi.go#L433), these are the reasons I see for being stuck in "Pending":
1. virt-launcher pod isn't scheduled
2. Topology hints for VMI aren't updated
3. DV is not provosioned / ready

**Review by iholder-redhat — COMMENTED** [review]
Thanks @machadovilaca! Looks great.
Left some comments

File: docs/runbooks/KubeVirtVMStuckInErrorState.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# KubeVirtVMStuckInErrorState
+
+## Meaning
+
+The `KubeVirtVMStuckInErrorState` alert means that a VirtualMachine has been in
```
[the remark this answers is not in the document]
Comment (machadovilaca) [reply docs/runbooks/KubeVirtVMStuckInErrorState.md:5]: done

**Comment by iholder-redhat** [comment]
Looks good from my perspective
/lgtm

**Comment by iholder-redhat** [comment]
Let's wait until the implementation is actually merged: https://github.com/kubevirt/kubevirt/pull/7918
/hold

**Comment by machadovilaca** [comment]
Actually for the tests to pass in the implementation PR we need to have the runbooks merged first
/unhold

**Pull request merged**